web-components/src/components/cards/ReviewCard/ReviewCard.Photo.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  CardMedia,
+  CardMediaProps,
+  useTheme,
+  useMediaQuery,
+} from '@mui/material';
+
+import Card from '../Card';
+
+const Photo: React.FC<CardMediaProps> = ({ src }) => {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
We might not need `useMediaQuery` here since it's slower to render than using plain media query.

Reply:
gone